CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.4.2](https://github.com/go-gitea/gitea/releases/tag/v1.4.2) - 2018-06-04
+* BUGFIXES
+  * Adjust z-index for floating labels (#3939) (#3950)
+  * add missing token validation on application settings page (#3976) #3978
+  * webhook and hook_task clean up (#4006)
+  * fix webhook bug of response info is not displayed in UI (#4023)
+  * fix writer cannot read bare repo guide (#4033) (#4039)
+  * Don't force due date to current time (#3830) (#4057)
+  * Fix wiki redirects (#3919) (#4065)
+  * fix attachment ENABLED (#4064) (#4066)
+  * Added deletion of an empty line at the end of file (#4054) (#4074)
+  * Use ResolveReference instead of path.Join (#4073)
+  * Fix #4081 Check for leading / in base before removing it (#4083)
+* MISC
+  * Update git vendor (#4059) (#4075)
```

Comment:
This should be moved under BUGFIXES with description `Respository's home page not updated after first push`

Also please upper case first letter in each sentence.

Reply:
Thanks for feedback @lafriks I've updated changelog accordingly.